pyodata/v2/service.py

```diff
@@ -284,7 +284,7 @@ def execute(self):
         # pylint: disable=assignment-from-none
         body = self.get_body()
 
-        headers = {} if self._headers is None else self._headers
+        headers = self.headers
```

Comment:
This one

Reply:
Here we should have `headers = dict(self.headers)` to make sure we don't polute someones dict.